packages/thyra/core/src/support/operator_solve/client_support/Thyra_DefaultBlockedTriangularLinearOpWithSolve_def.hpp

```diff
@@ -573,18 +573,26 @@ DefaultBlockedTriangularLinearOpWithSolve<Scalar>::solveImpl(
     &B = dyn_cast<const ProductMultiVectorBase<Scalar> >(B_in);
   ProductMultiVectorBase<Scalar>
     &X = dyn_cast<ProductMultiVectorBase<Scalar> >(*X_inout);
-  
+
+  bool converged = true;
```

Comment:
Instead of a simple `converged` flag, it would be better to use the functions [accumulateSolveStatusInit()](https://trilinos.org/docs/dev/packages/thyra/doc/html/structThyra_1_1SolveStatus.html#ab0ed482367e53ed438e2a6dbd3917848) and [accumulateSolveStatus](https://trilinos.org/docs/dev/packages/thyra/doc/html/structThyra_1_1SolveStatus.html#aca3dea7982de16447ca8c521ef659d3f).  That will do a better job on returning the correct solve status with useful information I think.

Reply:
OK, I wasn't aware of those.